_posts/2022-05-11-switch_statement_next_js.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: How to create switch statement inside rendering method in Next.js?
+categories: [Next.js]
+author_github: negativename
+author: Vladislav Kobenko
+---
+
+In case you want to create switch statement in rendering method in `Next.js` like this:
```

Comment:
```suggestion
In case you want to create a switch statement in rendering method in `Next.js` similar to the one below you can use on of the methods described in this post.
```